spec/integration/rendering_spec.rb

```diff
@@ -1,8 +1,10 @@
 RSpec.describe "Rendering" do
-  it "prints required params" do
-    output = `foo`
+  it "prints required params and banners" do
+    output = `foo 2>&1`
```

Comment:
Why do you redirect STDERR to STDOUT here? This is a reason why Travis fails, I think.

Reply:
Oh. My bad, I didn't remember I've added it and I thought it was already there 🤦‍♂️